internal/client/group-replication-client.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright 2024 The Kubernetes-CSI-Addons Authors.
```

Comment:
the filename is a little weird, it is in the `client` package, and also ends in `-client.go`.